src/docs/release/breaking-changes/platform-views-using-html-slots-web.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HTML slots to render platform views in the web
+description: iframes in Flutter web used to reload, because of the way some DOM operations were made. A change in the way Flutter web apps render Platform Views makes them much more stable (preventing iframe reloads, and other problems with video tags or forms potentially losing their state).
```

Comment:
Remove "much more". It wasn't stable and now it simply is :)

Reply:
Hah, yep. Removed!